server/tests/batch_comparison/test_batch_tallies.py

```diff
@@ -683,3 +683,28 @@ def test_batch_tallies_reprocess_after_manifest_reupload(
     )
 
     assert Jurisdiction.query.get(jurisdiction_ids[0]).batch_tallies is not None
+
+
+def test_batch_tallies_template_generation(
+    client: FlaskClient,
+    election_id: str,
+    jurisdiction_ids: List[str],
+    contest_ids: List[str],  # pylint: disable=unused-argument
+):
+    for user_type, user_email in [
+        (UserType.AUDIT_ADMIN, DEFAULT_AA_EMAIL),
+        (UserType.JURISDICTION_ADMIN, default_ja_email(election_id)),
+    ]:
+        set_logged_in_user(client, user_type, user_email)
+
+        rv = client.get(
+            f"/api/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/batch-tallies/template"
+        )
+        assert rv.status_code == 200
+        template_contents = rv.data.decode("utf-8")
+        assert template_contents == (
+            "Batch Name,candidate 1,candidate 2,candidate 3\r\n"
+            "Batch 1,0,0,0\r\n"
+            "Batch 2,0,0,0\r\n"
+            "Batch 3,0,0,0\r\n"
```

Comment:
Just realizing that I could take this a step further and, if a ballot manifest has been uploaded, pull actual batch names. But it might be confusing for the template to dynamically change like that.